luminoth/utils/config.py

```diff
@@ -33,9 +33,27 @@ def merge_into(new_config, base_config):
         if key not in base_config:
             raise KeyError('Key "{}" is not a valid config key.'.format(key))
 
+        # For Python2 and Python3 compatibility.
+        try:
+            unicode
+        except NameError:
+            # If 'unicode' isn't defined, set it as the type of a unicode
+            # string.
+            # TODO: unicode = str is causing an error later on in Python2,
+            # although this block should only run on Python3 and `type(u'u')`
+            # should be `str` in that case, anyway.
+            # Possible Python2 overoptimization bug?
+            unicode = type(u'u')
         # Since we already have the values of base_config we check against them
         if (base_config[key] is not None and
-           type(base_config[key]) is not type(value)):
+            # Allow all values to be None or False.
```

Comment:
Find a better way to write this, especially when needing comments between conditions.
Maybe use another function.